src/js/shield_defs.js

```diff
@@ -4126,6 +4130,16 @@ export function loadShields() {
     "Western Kentucky Pkwy": "WK",
   };
 
+  shields["US:CT:Parkway"].refsByWayName = {
+    "Merritt Parkway": "MP",
+  };
+
+  shields["US:CT:Parkway"].overrideByRef = {
+    MP: {
+      spriteBlank: "shield_us_ct_parkway_merritt",
+    },
+  };
+
```

Comment:
Syntax-wise, this seems a little kludgy. Is there any way we can say something more simple like:

```json
"Merritt Parkway": "shield_us_ct_parkway_merritt"
```

Reply:
I added an `overrideByWayName` property to enable this. I left the code that allows `refsByWayName` and `overrideByRef` to work together since there are still cases that might be useful.